pydantic/json_schema.py

```diff
@@ -1609,13 +1609,13 @@ def json_schema(self, schema: core_schema.JsonSchema) -> JsonSchemaValue:
         Returns:
             The generated JSON schema.
         """
+        content_core_schema = schema['schema'] if 'schema' in schema else core_schema.any_schema()
```

Comment:
I would like the following code more but it will always call `core_schema.any_schema()`

```suggestion
        content_core_schema = schema.get('schema', core_schema.any_schema())
```

It would be nice to have a constant like `core_schema.DEFAULT_SCHEMA` that will hold this value and allow the above to be as efficient as `if..in`.